timemachine/potentials/chiral_restraints.py

```diff
@@ -2,6 +2,10 @@
 import jax.numpy as jnp
 
 
+def normalize(x):
```

Comment:
maybe rename to `unit_vector(x)`

Reply:
This would be less clear in my opinion (noun vs. verb), and I'm not aware of an ambiguity with the current name ("normalize" doesn't seem easily confused with anything else in this context).

Can change this name in a subsequent pass if needed.